dbclient/jdbc/src/main/java/io/helidon/dbclient/jdbc/JdbcDbClient.java

```diff
@@ -209,7 +209,7 @@ public T apply(Throwable t) {
 
     @Override
     public Single<Void> ping() {
```

Comment:
This method should probably be deprecated, as the health check may be either DML or Query.

Reply:
Yes, this ping looks like some workaround or shortcut. User should use health check buillder/API to call proper statement.